application/backend/src/main/java/pl/ds/bulma/components/models/BlockComponent.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2022 Dynamic Solutions
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pl.ds.bulma.components.models;
+
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.Model;
+
+@Model(adaptables = Resource.class, defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL)
+public class BlockComponent {
```

Comment:
We don't need a model class if we don't have anything there.

Reply:
Absolutely makes sense. I just created this, because I used FooterComponent as an example, and there it's created even though it's empty but maybe there's a special consideration behind it. Anyways, I deleted it.

Reply:
I missed that we have such an empty model. We will have to clean up this later.